Formula/gcc.rb

```diff
@@ -63,15 +64,27 @@ def version_suffix
     end
   end
 
+  resource "bootstrap_gcc" do
+    url "https://phoenixnap.dl.sourceforge.net/project/gnuada/GNAT_GCC%20Mac%20OS%20X/11.1.0/native/gcc-11.1.0-x86_64-apple-darwin15.pkg"
+    sha256 "d947b5db0576cb62942e5ce61f3ef53fb679f07b1adff7a4c0fa19a5e72a9532"
+  end
+
   def install
+    resource("bootstrap_gcc").stage do
+      system "pkgutil", "--expand-full", "gcc-11.1.0-x86_64-apple-darwin15.pkg", buildpath/"bootstrap_gcc"
+    end
+    bootstrap_gcc_prefix = buildpath/"bootstrap_gcc/gcc-11.1.0-x86_64-apple-darwin15.pkg/Payload"
+    ENV["CC"] = bootstrap_gcc_prefix/"bin/gcc"
+    ENV["CXX"] = bootstrap_gcc_prefix/"bin/g++"
+    ENV.append_to_cflags "--sysroot=#{MacOS.sdk_path}"
+
     # GCC will suffer build errors if forced to use a particular linker.
     ENV.delete "LD"
 
     # We avoiding building:
-    #  - Ada, which requires a pre-existing GCC Ada compiler to bootstrap
     #  - Go, currently not supported on macOS
     #  - BRIG
-    languages = %w[c c++ objc obj-c++ fortran]
+    languages = %w[ada c c++ objc obj-c++ fortran]
```

Comment:
What's the difference in bottle size / installed disk space, if we decide to ship Ada?

Reply:
357MB to 620MB